hometask7

```diff
@@ -0,0 +1,109 @@
+# Task 1
+import json
+
+with open('json_phonebook.json') as file:
```

Comment:
Your file will be opened during the whole program execution. This is not how it should work. Open your file only in places where you need it.